src/providers/anthropic/chatComplete.ts

```diff
@@ -394,7 +410,8 @@ export const AnthropicChatCompleteResponseTransform: (
           },
           index: 0,
           logprobs: null,
-          finish_reason: response.stop_reason,
+          finish_reason:
+            getFinishReason(response.stop_reason) || response.stop_reason,
```

Comment:
This should just be `getFinishReason(response.stop_reason)`, Otherwise it defeats the purpose of strictly typing the response